contracts/vaults/MetaStable2TokenAuraVault.sol

```diff
@@ -185,5 +188,24 @@ contract MetaStable2TokenAuraVault is MetaStable2TokenVaultMixin {
             totalBPTSupply: IERC20(context.poolContext.basePool.pool).totalSupply()
         });
     }
+
+    function migrateAura() external onlyNotionalOwner {
+        uint256 oldAmount = OLD_REWARD_POOL.balanceOf(address(this));
+        
+        bool success = OLD_REWARD_POOL.withdrawAndUnwrap(oldAmount, true);
+        if (!success) revert Errors.UnstakeFailed();
+
+        IERC20(address(BALANCER_POOL_TOKEN)).checkApprove(address(AURA_BOOSTER), type(uint256).max);
```

Comment:
We approve max here because we also need to have AURA_BOOSTER have future approvals for additional deposits?

Reply:
yeah, we did the same thing during contract initialization for the old booster